codec-http2/src/main/java/io/netty/handler/codec/http2/InboundHttp2ToHttpAdapter.java

```diff
@@ -124,10 +124,10 @@ public void onStreamRemoved(Http2Stream stream) {
     /**
      * Set final headers and fire a channel read event
      *
-     * @param ctx The context to fire the event on
-     * @param msg The message to send
+     * @param ctx     The context to fire the event on
+     * @param msg     The message to send
      * @param release {@code true} to call release on the value if it is present. {@code false} to not call release.
-     * @param stream the stream of the message which is being fired
+     * @param stream  the stream of the message which is being fired
```

Comment:
This aligned parameter description style is not something we do anymore. Can you turn it off in your editor and reformat the class?

Reply:
Fixed